cluster/cluster_client.ml

```diff
@@ -0,0 +1,14 @@
+
+open Xcp_client
```

Comment:
In line with comments in new clients, please don't open this globally, but add `let open Xcp_client` inside the `rpc` function body